src/Airy/cairy.jl

```diff
@@ -372,20 +410,57 @@ end
     return A, B, C, D
 end
 
+@generated function airyaix_levin(x::Complex{T}, ::Val{N}) where {T <: Union{Float32, Float64}, N}
+    :(
+        begin
+            xsqr = sqrt(x)
+            out = zero(typeof(x))
+            t = GAMMA_ONE_SIXTH(T) * GAMMA_FIVE_SIXTHS(T) / 4
+            a = inv(4*xsqr*x)
+
+            l = @ntuple $N i -> begin
+                out += t
+                t *= -3 * a * (i - 5//6) * (i - 1//6) / i
```

Comment:
`t *= a * (i - 5//6) * muladd(1/2, i,  -1)`

Reply:
I think it would have to be `t *= a * (i - 5//6) * muladd(1/2, 1/i,  -3)` but on a quick test it wasn't as accurate would have to check whole range though to be sure.

Reply:
wolfram suggests `a*(3 - 3*i - (5/12)/i)`

Reply:
Seems slightly more accurate will make that change.

Reply:
Also has advantage of one complex multiply instead of perhaps a complex and two scalar multiples depending if the compiler rearranged that at all